test/micro/org/openjdk/bench/vm/compiler/FluidSBBench.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
@shipilev I copied this benchmark from your RFE. Is this copyright notice correct?

Reply:
Yes.